### DISCUSSION TITLE
How to generate .attrs file manually ?

### DISCUSSION THREAD

**Discussion by hungcrush** [pocketbase/pocketbase#2384] [Q&A] [answered]
I'm converting my PHP project to Pocketbase, I copy all my storage files to pb_data and clone data without upload file.. Everything is ok, but I realize it can not create thumbnail cause it can't get Attributes.

So could you give me some ideas how to generate .attrs file again ?

Thank you so much.

**Comment by ganigeorgiev** [comment, marked as answer]
I don't think there is automated way without reupload.

If you have an import script that recreates the PocketBase storage files structure, you can create the `.attrs` sidecar file on your own for each entry. The minimal file content should be something like:

```json
{"user.content_type":"image/png"}
```
_(you can change `png` based on the imported file extension)_

**Reply by hungcrush** [reply]
Thank you. I got it 😬
